wicketd/src/bin/wicketd.rs

```diff
@@ -140,5 +153,22 @@ async fn do_run() -> Result<(), CmdError> {
                 .await
                 .map_err(|err| CmdError::Failure(anyhow!(err)))
         }
+        Args::RefreshConfig { config_file_path, address } => {
+            let config = Config::from_file(&config_file_path)
+                .with_context(|| {
+                    format!("failed to parse {}", config_file_path.display())
+                })
+                .map_err(CmdError::Failure)?;
+
+            let log = config
+                .log
+                .to_logger("wicketd")
+                .context("failed to initialize logger")
+                .map_err(CmdError::Failure)?;
+
+            Server::refresh_config(log, address)
+                .await
+                .map_err(CmdError::Failure)
```

Comment:
Could you put in some explicit documentation here about exit codes? I believe this exits with either 1 or 2, which is fine because it doesn't collide with any of the exit codes in https://illumos.org/man/7/smf_method.